scripts/piper/generate_lexicon_zh.py

```
@@ -0,0 +1,64 @@
+#!/usr/bin/env python3
+# Copyright    2026  Xiaomi Corp.        (authors: Fangjun Kuang)
+
+# It may take 100 minutes to generate the lexicon.
+#
+# You can download a pre-generated one from
+# https://github.com/k2-fsa/sherpa-onnx/releases/tag/tts-models
+
+from typing import List, Tuple
+
+from pypinyin import load_phrases_dict, phrases_dict, pinyin_dict
```

Flake8 (7.3.0)

[error] 'pypinyin.load_phrases_dict' imported but unused

(F401)